src/registries/npm-registry.js

```diff
@@ -130,7 +130,7 @@ export default class NpmRegistry extends Registry {
     const possibles = [
       [false, path.join(this.cwd, filename)],
       [true, this.config.userconfig || path.join(userHome, filename)],
-      [false, path.join(getGlobalPrefix(), filename)],
+      [false, path.join(getGlobalPrefix(), 'etc', filename.slice(1))],
```

Comment:
Actually to be properly correct it should probably take the RC name without the leading `.` and prefix it, rather than slicing it off.  If we're going to do this properly I can try and take a look tomorrow at filling it out a bit.

I don't know anything about the Windows install of this or NPM though, so someone else will need to check that aspect of it.